templates/worker-sites-react/package.json

```diff
@@ -27,8 +27,8 @@
 	},
 	"dependencies": {
 		"@cloudflare/kv-asset-handler": "workspace:*",
-		"react": "^16.9.0",
-		"react-dom": "^16.9.0"
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0"
```

Comment:
I don't think we have any tests of this template.
Is it definitely going to work with a big major bump like this?

Reply:
tested, works fine